### PR TITLE
align tombstone request with tombstone metadata

### PR DESCRIPTION
**Pull request by southeo**
updates tombstone request for handle api
We don't make pojos from this schema, it's just for completeness

### CODE REVIEW

File: data-model/fdo-profile/tombstone/1.0.0/tombstone-request.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                        "ods:isDuplicateOf"
+                      ]
+                    }
+                  },
+                  "additionalProperties": false
+                },
+                "additionalProperties": false,
+                "required": [
+                  "tombstoneText"
+                ]
```
Comment (samleeflang) [inline data-model/fdo-profile/tombstone/1.0.0/tombstone-request.json:52]: There is no tombstoneText here? Maybe `pid` should be required at least, so if a relatedPid is there we at least need the pid

File: data-model/fdo-profile/tombstone/1.0.0/tombstone-request.json [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+              }
+            }
+          },
+          "additionalProperties": false,
+          "required": [
+            "tombstoneText"
```
Comment (samleeflang) [inline data-model/fdo-profile/tombstone/1.0.0/tombstone-request.json:58]: Not sure if we will always have a tombstoneText, although we could probably always put something in, even if it is not that informative

**Review by samleeflang — COMMENTED** [review]
:open_file_folder:

File: data-model/fdo-profile/tombstone/1.0.0/tombstone-request.json (hunk 1 above)
Comment (southeo) [reply data-model/fdo-profile/tombstone/1.0.0/tombstone-request.json:52]: tombstoneText is the first field in `attributes`
also, the pid is provided in the `id` field

File: data-model/fdo-profile/tombstone/1.0.0/tombstone-request.json (hunk 2 above)
Comment (southeo) [reply data-model/fdo-profile/tombstone/1.0.0/tombstone-request.json:58]: We can discuss if it still needs to be required for the pid profile on our fdo meeting, but tombstone text is required for [ods tombstone attributes](https://schemas.dissco.tech/schemas/fdo-type/shared-model/0.3.0/tombstone-metadata.json), so we'll always have something

Comment (samleeflang) [reply data-model/fdo-profile/tombstone/1.0.0/tombstone-request.json:58]: ah ok, good.

**Review by samleeflang — APPROVED** [review]
:womans_hat:

**Pull request merged**